tests/test_fields.py

```diff
@@ -1773,8 +1773,31 @@ class TestDurationField(FieldValues):
     field = serializers.DurationField()
 
 
-# Choice types...
+class TestNoOutputFormatDurationField(FieldValues):
+    """
+    Values for `TimeField` with a no output format.
+    """
+    valid_inputs = {}
+    invalid_inputs = {}
+    outputs = {
+        datetime.timedelta(1): datetime.timedelta(1)
+    }
+    field = serializers.DurationField(format=None)
 
+
+class TestISOOutputFormatDurationField(FieldValues):
+    """
+    Values for `TimeField` with a custom output format.
+    """
+    valid_inputs = {}
+    invalid_inputs = {}
```

Comment:
Would be nice to add test coverage for parsing duration in ISO format, as I would expect it to work both ways

Reply:
Currently the [`parse_duration`](https://docs.djangoproject.com/en/5.2/ref/utils/#django.utils.dateparse.parse_duration) method of django, which is used inside `DurationField`, already parses ISO-8601 format and the format specific to postgres by its own.

This could be added to the `TestDurationField` testcase.

Where do you prefer these tests to be added?

PS: 
From this point of view the current error message `"Use one of these formats instead: [DD] [HH:[MM:]]ss[.uuuuuu]"` is not (completely) right, however changing it would be a breaking change.

Reply:
I would not consider extending the message a breaking change, as it's obvious from its wording that it might get extended and you shouldn't depend on it remaining literally unchanged.

Reply:
I'm not sure where that an extension of this error message should be considered a breaking change; the wording "one of" already hints that the application should not rely the one specific format being given in the message.